quit/basicfunction.py

```diff
@@ -0,0 +1,121 @@
+import typing
+from collections.abc import Iterable
+
+import numpy as np
+from scipy import linalg
+
+
+def ket(psi: Iterable[complex]) -> np.ndarray:
+    """Create a ket |psi> as a vertical vector.
+
+    :param ket: Sequence of complex numbers.
+    :return: A vertical vector as numpy array.
+    """
+    return np.reshape(np.asarray(psi), (-1, 1))
+
+
+def basis(dim: int) -> Iterable[np.ndarray]:
```

Comment:
It seems to me that it would be more useful if we could construct i-th basis vector instead of constructing all of them at once.

WDYT about changing the signature to
```
def basis(index: int, dim: int) -> np.ndarray
```
We should make sure that the order of arguments is the same as in `qutip`.